src/EPR.Payment.Facade/wwwroot/index.html

```diff
@@ -310,7 +310,7 @@
             }
 
             try {
-                const response = await fetch(`/api/v1/producers-fees?regulator=${regulator}`, {
+                const response = await fetch(`/api/v1/producers-fees/resubmission?regulator=${regulator}`, {
```

Comment:
the correct route is ; ...../v1/producer/resubmission-fee

Reply:
it covered with another ticket